l1-contracts/test/merkle/Merkle.t.sol

```diff
@@ -5,17 +5,41 @@ pragma solidity >=0.8.18;
 import {Test} from "forge-std/Test.sol";
 
 import {NaiveMerkle} from "./Naive.sol";
+import {Merkle} from "../../src/core/libraries/Merkle.sol";
+import {Errors} from "../../src/core/libraries/Errors.sol";
 import {FrontierMerkle} from "./../../src/core/messagebridge/frontier_tree/Frontier.sol";
 
-contract MerkleTest is Test {
+abstract contract MerkleLibWrapper {
```

Comment:
I think readability would be better here if you used the helper approach like what we used for example in TxsDecoderHelper. Having this abstract contract and then inheriting from it in the test seems less readable to me than having test code and the library separated in different contracts.